matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/terms/TermsService.kt

```diff
@@ -19,7 +19,12 @@ package org.matrix.android.sdk.api.session.terms
 interface TermsService {
     enum class ServiceType {
         IntegrationManager,
-        IdentityService
+        IdentityService,
+
+        /**
+         * It's only possible to use this value for [getTerms]
+         */
+        Homeserver
```

Comment:
do you think it's worth having a separate `getHomeserverTerms` to avoid potential misunderstandings with this api?

Reply:
I though about it, will iterate.

Reply:
49721ed14d19c2dd86276e16e3c1bcf3a5d79884